Automattic/wp-calypso
# Home: Move notices to separate components

#### Changes proposed in this Pull Request

* Moves all Home notices into their own components. 
* It also consolidates all of them into the same celebrate format we started to use as a pattern for this kind of notices.
* The introductory welcome banner added for users that have never been redirected there (https://github.com/Automattic/wp-calypso/pull/39557) has been removed here in order to clean up a bit further the codebase. Likely, most of active users have already seen that banner so it should be safe to remove it

Before | After
--- | ---
<img width="971" alt="Screen Shot 2020-03-19 at 15 14 25" src="https://user-images.githubusercontent.com/1233880/77092922-9463b100-6a0a-11ea-89df-0e993730f00f.png"> | ![image](https://user-images.githubusercontent.com/1233880/77092949-9c235580-6a0a-11ea-81db-cb8b37bbac5c.png)
![image](https://user-images.githubusercontent.com/1233880/77093081-c83ed680-6a0a-11ea-8240-32adfe154542.png) | ![image](https://user-images.githubusercontent.com/1233880/77093089-cb39c700-6a0a-11ea-8073-e3f91d8167ee.png)
![image](https://user-images.githubusercontent.com/1233880/77093105-d2f96b80-6a0a-11ea-9087-b1676a678578.png) | <img width="967" alt="Screen Shot 2020-03-20 at 11 45 40" src="https://user-images.githubusercontent.com/1233880/77163813-34731600-6aaf-11ea-9e86-0afb92f734c8.png">
<img width="967" alt="Screen Shot 2020-03-20 at 13 41 58" src="https://user-images.githubusercontent.com/1233880/77164478-adbf3880-6ab0-11ea-87b3-112d054a5679.png"> | <img width="976" alt="Screen Shot 2020-03-20 at 13 42 30" src="https://user-images.githubusercontent.com/1233880/77164492-b748a080-6ab0-11ea-8b0c-d07837dd8888.png">


#### Testing instructions

* Create a site.
* Make sure Home displays a congratulatory message for the new site created.
* Launch the site.
* Make sure Home displays a congratulatory message for the launched site.
* Complete the checklist.
* Make sure Home displays a congratulatory message for the site setup being complete.
* Reload the page and make sure the checklist complete notice is gone.
* Migrate a site to WP.com (switch to a Simple site, go to Tools > Import, select WordPress, enter the URL of a Jetpack site and select Everything).
* Make sure Home displays a congratulatory message for the new imported site.
